tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/05_deleteAbandonedCarts.js

```diff
@@ -0,0 +1,107 @@
+require('module-alias/register');
+
+// Import expect from chai
+const {expect} = require('chai');
+
+// Import utils
+const helper = require('@utils/helpers');
+const testContext = require('@utils/testContext');
+
+// Import common tests
+const loginCommon = require('@commonTests/BO/loginBO');
+const {createShoppingCart} = require('@commonTests/FO/createShoppingCart');
+
+// Import BO pages
+const dashboardPage = require('@pages/BO/dashboard');
+const shoppingCartsPage = require('@pages/BO/orders/shoppingCarts');
+
+// Import data
+const {DefaultCustomer} = require('@data/demo/customer');
+const {Products} = require('@data/demo/products');
+
+const baseContext = 'functional_BO_orders_shoppingCarts_deleteAbandonedCart';
+
+let browserContext;
+let page;
+let numberOfShoppingCarts;
+
+const orderByCustomerData = {
+  customer: DefaultCustomer,
+  product: Products.demo_1,
+  productQuantity: 1,
+};
+
+/*
+Pre-condition:
+- creat shopping cart from FO
+Scenario:
+- Filter shopping carts by non ordered
+- delete them if exist
+ */
+
+// Pre-condition: Create 1 order in FO
+createShoppingCart(orderByCustomerData, `${baseContext}_preTest_1`);
```

Comment:
The pre-condition should be inside the describe

Reply:
Done